js/utils/helpers.js

```diff
@@ -1924,8 +1924,12 @@
                                         }
                                         break;
                                     default:
+                                        if (attr[0] == "width") {
```

Comment:
To accomplish getting the button to pull right to the same width as the iframe, I capture the width value for later use as part of the div that wraps the figcaption and fullscreen button (when caption is at top).

Reply:
What you've done makes sense, Just make sure you've documented it somewhere. You should explain what is the expected behavior (where it should be aligned) and why you did it this way, so later we won't be confused by it.